docker-compose.local.yml

```diff
@@ -30,10 +32,10 @@ services:
     - postgres
     restart: always
     environment:
-      HASURA_GRAPHQL_DATABASE_URL: postgres://hello_zm:hello_zm@postgres:5432/hello_zm_dev
-      HASURA_GRAPHQL_ENABLE_CONSOLE: "true" # set to "false" to disable console
-      HASURA_GRAPHQL_ENABLED_LOG_TYPES: startup, http-log, webhook-log, websocket-log, query-log
-      HASURA_GRAPHQL_ADMIN_SECRET: dev_only_password
+      - HASURA_GRAPHQL_ADMIN_SECRET
```

Comment:
IMHO it may be a) not needed b) harmful if overrides `.env`

Reply:
Docker isn't automatically passing all env variables to all containers